instrumentation/elastic-opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/helpers.py

```diff
@@ -292,48 +290,61 @@ def _send_log_events_from_messages(
         if message["role"] == "system" or message["role"] == "developer":
             if message["role"] == "developer":
                 body["role"] = message["role"]
-            event = Event(name=EVENT_GEN_AI_SYSTEM_MESSAGE, body=body, attributes=attributes)
-            event_logger.emit(event)
+            # keep compat on the exported attributes with Event
```

Comment:
IIUC this is about event.name. AFAIU this is the long term direction and is reasonable so we probably don't need a comment since it seems like we are doing something special